asteroid/masknn/tac.py

```diff
@@ -0,0 +1,56 @@
+import torch
+import torch.nn as nn
+from asteroid.masknn import activations, norms
+
+
+class TAC(nn.Module):
+    def __init__(self, input_dim, hidden_dim=384, activation="prelu", norm_type="gLN"):
+        super().__init__()
+
+        self.hidden_dim = hidden_dim
+        self.input_tf = nn.Sequential(
+            nn.Linear(input_dim, hidden_dim), activations.get(activation)()
+        )
```

Comment:
Adding a `self.activation = activations.get(activation)`  and `self.norm = norm_type` could save space and makes things a bit clearer

Reply:
but activation must be different for each MLP block